recipe/meta.yaml

```diff
@@ -35,21 +42,63 @@ requirements:
     # need a CDT for libudev for cos6, proper package available for cos7 (below)
     - {{ cdt('libudev-devel') }}  # [linux and cdt_name=='cos6']
   host:
+    - python
     - libudev  # [linux and cdt_name!='cos6']
     - libusb
   run:
+    - python
     - __cuda  # [cuda_compiler_version != "None"]
 
-test:
-  commands:
-    - test -f ${PREFIX}/include/librealsense2/rs.hpp  # [not win]
-    - test -f ${PREFIX}/lib/librealsense2.so  # [linux]
-    - test -f ${PREFIX}/lib/librealsense2.dylib  # [osx]
-    - test -f ${PREFIX}/lib/cmake/realsense2/realsense2Config.cmake  # [not win]
-    - if exist %PREFIX%\\Library\\include\\librealsense2\\rs.hpp (exit 0) else (exit 1)  # [win]
-    - if exist $PREFIX$\\Library\\lib\\realsense2.lib (exit 0) else (exit 1)  # [win]
-    - if exist $PREFIX$\\Library\\bin\\realsense2.dll (exit 0) else (exit 1)  # [win]
-    - if exist %PREFIX%\\Library\\lib\cmake\\realsense2\\realsense2Config.cmake (exit 0) else (exit 1)  # [win]
+
+outputs:
+  - name: librealsense
+  - name: librealsense2
```

Comment:
This is the name of the package that depends on both the C++ library (librealsense) and the python package (pyrealsense2, with the 2 for consistency with PyPI https://pypi.org/project/pyrealsense2/). I think it would be confusing to call this `librealsense2` . Probably for the time being we can just remove this output until it is actually required?

Reply:
I have to admit I don't fully understand how opencv's build works, but it seemed to me like librealsense2 is should only be the c++ package (which also builds python therefore being tied as a python version), librealsense itself is then the version pining between pyrealsense2 & librealsense2 to the same version to avoid dynamic linking conflicts. 

If that is the case I don't mind making just `librealsense` <-- `pyrealsense2`

Reply:
I think that `{{ pin_subpackage('librealsense', exact=True) }}` in pyrealsense2 already ensures that the proper version is used, so I think we can just do `librealsense` <-- `pyrealsense2`.